cmd/vm_start.go

```diff
@@ -23,24 +22,28 @@ var vmStartCmd = &cobra.Command{
 			return err
 		}
 
-		errs := []error{}
+		tasks := make([]task, 0, len(args))
 		for _, v := range args {
-			if err := startVirtualMachine(v, rescueProfile); err != nil {
-				errs = append(errs, fmt.Errorf("could not start %q: %s", v, err))
+			vm, err := getVirtualMachineByNameOrID(v)
+			if err != nil {
+				return err
 			}
-		}
+			state := (string)(egoscale.VirtualMachineStopped)
```

Comment:
nitpick: I would rename this "stoppedState"

Reply:
in this case the variable is only used one line after, when you read `egoscale.VirtualMachineStopped` you understand clearly.
https://talks.golang.org/2014/names.slide